OracleIdentityGovernance/README.md

```diff
@@ -0,0 +1,128 @@
+Oracle Identity Governance (OIG) on Docker
+==========================================
+
+
+## Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites for OIG on Docker](#prerequisites-for-oig-on-docker)
+3. [Prerequisites for OIG on Kubernetes](#prerequisites-for-oig-on-kubernetes)
+4. [Installing the OIG Docker Image](#installing-the-oig-docker-image)
+5. [OIG Docker Container Configuration](#oig-docker-container-configuration)
+6. [OIG Kubernetes Configuration](#oig-kubernetes-configuration)
+7. [Appendix A: Using an Oracle Database Docker Image for OIG](#appendix-a-using-an-oracle-database-docker-image-for-oig)
+
+## Introduction
+
+This project offers Dockerfiles and scripts to build and configure an Oracle Identity Governance image based on 12cPS4 (12.2.1.4.0) release.
+Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+## Prerequisites for OIG on Docker
+
+The following prerequisites are required for building OIG Docker images:
+
+* A working installation of Docker 18.03 or later
+* Docker-compose 1.25.4 or higher
```

Comment:
Why is `docker-compose` required for building? Note that while Oracle builds and supports Oracle Container Runtime for Docker on Oracle Linux 7 and Podman on Oracle Linux 8, we do not support any release of `docker-compose`, so making it a mandatory requirement is not recommended.

Reply:
OIG depends on SOA. SOA which is already released is using docker-compose. So we have extended on similar lines